WcaOnRails/app/models/competition.rb

```diff
@@ -40,7 +40,13 @@ class Competition < ActiveRecord::Base
   validate :must_have_at_least_one_event, if: :isConfirmed?
   def must_have_at_least_one_event
     if events.length == 0
-      errors.add(:eventSpecs, "Competition must have at least one event")
+      errors.add(:Events, "Competition must have at least one event")
```

Comment:
It's the label, I was trying to make it turn red...didn't work.
I'll change back to eventSpecs